content/posts/newsletter/2020-08-08.md

```diff
@@ -34,6 +34,13 @@ You do not need to use a formal tone, be passionate about why this motivates you
 
 Monal is organizing translations for the app on [Weblate](https://hosted.weblate.org/engage/monal/) now. 27 languages are waiting for excellent and native speakers!
 
+There have been UI and file sharing improvements on the "[Salut à Toi](https://salut-a-toi.org/)" project:
+
+- Libervia, the web frontend, has seen a major UI revamp, with a new theme based on Bulma CSS framework. The [W31 progress note](https://www.goffi.org/b/n9nzn3ySF7Lv4pYBpTqYtP/progress-note) also explains how Nunjucks is used in the browser to use the same templates as in the backend (Jinja 2).
+- SàT's included file sharing component now implements [HTTP File Upload](https://xmpp.org/extensions/xep-0363.html), which completes the [Jingle File Transfer](https://xmpp.org/extensions/xep-0234.html) method. Files uploaded to this component from any XMPP client will appear in SàT's file sharing UI, and can be deleted from there (or with [Ad-Hoc Commands](https://xmpp.org/extensions/xep-0050.html)).
```

Comment:
to be consistent across the newsletter, I suggest:
`[HTTP File Upload](https://xmpp.org/extensions/xep-0363.html)` => `[XEP-0363](https://xmpp.org/extensions/xep-0363.html) (HTTP File Upload)`
`[Jingle File Transfer](https://xmpp.org/extensions/xep-0234.html)` => `[XEP-0234](https://xmpp.org/extensions/xep-0234.html) (Jingle File Transfer)`
`[Ad-Hoc Commands](https://xmpp.org/extensions/xep-0050.html)` => `[XEP-0050](https://xmpp.org/extensions/xep-0050.html) (Ad-Hoc Commands)`